### PR TITLE
Initial attempt to split storybook stories into separate snapshots

### PR DESCRIPTION
**Pull request by brentswisher**
Work in Progress

## Description
Updates the storybook add-on story shots to save snapshots in separate stories for each component.

Due to what appears to be an [unfixed bug](https://github.com/storybookjs/storybook/issues/9774#issuecomment-583691442) in story shots, each story will also have to export `parameters: { fileName: __filename },` 

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: storybook/test/index.js [hunk 1]
```diff
@@ -25,24 +25,6 @@ afterAll( () => {
...
-				);
-			}
-			if ( story.kind === 'Components/Popover' ) {
-				const parentElement = document.createElement( 'div' );
-				parentElement.appendChild( currentElement );
-			}
-			return currentElement;
-		},
-	} ) ),
+	integrityOptions: { cwd: __dirname }, // it will start searching from the current directory
```
Comment (gziolo) [inline storybook/test/index.js:28]: This mocking logic is essential to make tests pass when components use `createRef`.

**Comment by gziolo** [comment]
Interesting exploration 👍

File: storybook/test/index.js (hunk 1 above)
Comment (brentswisher) [reply storybook/test/index.js:28]: Yes, I was waiting to see which direction #21485 went before spending more time getting all the tests running correctly. I didn't want to spend the time if the decision is to drop storyshots altogether.

Comment (gziolo) [reply storybook/test/index.js:28]: we can always make those snapshots more useful but adding additional assertions like running a11y checks on the generated content.

**Comment by gziolo** [comment]
Should it be closed now that we are removing StoryShots? 😃

**Comment by brentswisher** [comment]
Yes, was just waiting for that to be merged 👍

**Pull request closed without merging**